src/pages/ForgotPassword/ForgotPassword.tsx

```diff
@@ -24,15 +58,25 @@ const ForgotPassword = () => {
         <AuthSub>
           No worries! <br /> Enter your email and we will send you a reset
```

Comment:
One suggestion - add a period after "reset". Since we have punctuation after "No worries!", I think we need it here, too.